### PR TITLE
Setting another environment variable in AWS + Updating README

### PR DESCRIPTION
**Pull request by ghostcodekc**
Added environment variable 'CLOUD_NAME' in Lambda correlating with the Cloudinary cloud name, so it's easier for user setup. 

Added in information about locales for English for MomentJS.

### CODE REVIEW

File: warlog.js [hunk 1]
```diff
@@ -41,45 +41,52 @@ exports.handler = async (event, context) => {
...
+            const allFriendlies = playerBattlesJson.filter(battle => battle.type === 'clanMate').length;
+
             const text =
                 `${battle.winner >= 1 ? 'Victory! :raised_hands:' : 'Loss :crying_cat_face:'}\n` +
                 `${battle.team[0].name} vs ${battle.opponent[0].name} at ` +
                 `${moment
                     .unix(battle.utcTime)
-                    .locale('nb')
-                    .tz('Europe/Oslo')
+                    .locale('en')
```
Comment (olemartin) [inline warlog.js]: You should have this as environment variables as well

Comment (olemartin) [reply warlog.js]: these.. :)

File: warlog.js [hunk 2]
```diff
@@ -97,6 +104,20 @@ exports.handler = async (event, context) => {
     return results;
 };
 
+const equalDeck = (warDeck, otherDecks) => {
```
Comment (olemartin) [inline warlog.js]: I don't understand why this is diffing, but I figure it out

**Review by olemartin — COMMENTED** [review]
Great work! Would be great if you could look at my comments

**Comment by ghostcodekc** [comment]
Sorry it took me so long. I changed the time zone and language locale to environment variables in AWS. :)

File: warlog.js (hunk 2 above)
Comment (ghostcodekc) [reply warlog.js]: I'm not sure, I downloaded the current master branch when I worked on the environment variables. Let me know if you figure something out :)

**Comment by olemartin** [comment]
Thank you so much!

I had some other changes going, and I couldn't figure out how to merge these two changes in a good way. So I copied the changes from your pull request and added it to my commit.
So your changes are not lost even if I close the pull request.
Thanks again 💯 👍

**Pull request closed without merging**